includes/apple-exporter/builders/class-text-styles.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
Please check tabbing between the various php files, some seem to have spaces, some tabs, might just be how it looks in github, /shrug